test/plugin/scenarios/grpc-scenario/config/expectedData.yaml

```diff
@@ -15,239 +15,524 @@
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 23
+      componentName: ''
+      isError: false
+      spanType: Entry
+      peer: ''
+      peerId: 0
+      refs:
+        - {parentEndpointId: 0, parentEndpoint: /grpc-scenario/case/grpc-scenario, networkAddressId: 0,
+           entryEndpointId: 0, refType: CrossProcess, parentSpanId: 5, parentTraceSegmentId: not null,
+           parentServiceInstanceId: 1, networkAddress: '127.0.0.1:18080', entryEndpoint: /grpc-scenario/case/grpc-scenario,
+           entryServiceInstanceId: 1}
+  - segmentId: not null
+    spans:
+    - operationName: GreeterBlockingError.sayHello/server/Response/onClose
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: RPCFramework
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 23
+      componentName: ''
+      isError: true
+      spanType: Local
+      peer: ''
+      peerId: 0
+      tags:
+      - { key: status_code, value: UNKNOWN }
+      logs:
+      - logEvent:
+        - { key: event, value: error }
+        - { key: error.kind, value: java.lang.Exception }
+        - { key: message, value: '' }
+        - key: stack
+          value: not null
+      refs:
+      - {parentEndpointId: 0, parentEndpoint: GreeterBlockingError.sayHello, networkAddressId: 0,
+         entryEndpointId: 0, refType: CrossThread, parentSpanId: 0, parentTraceSegmentId: not null,
+         parentServiceInstanceId: 1, networkAddress: '', entryEndpoint: /grpc-scenario/case/grpc-scenario,
+         entryServiceInstanceId: 1}
+    - operationName: GreeterBlockingError.sayHello/server/Request/onComplete
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: RPCFramework
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 23
+      componentName: ''
+      isError: false
+      spanType: Local
+      peer: ''
+      peerId: 0
+      refs:
+      - {parentEndpointId: 0, parentEndpoint: GreeterBlockingError.sayHello, networkAddressId: 0,
+         entryEndpointId: 0, refType: CrossThread, parentSpanId: 0, parentTraceSegmentId: not null,
+         parentServiceInstanceId: 1, networkAddress: '', entryEndpoint: /grpc-scenario/case/grpc-scenario,
+         entryServiceInstanceId: 1}
+  - segmentId: not null
+    spans:
+    - operationName: Greeter.sayHello
+      operationId: 0
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: RPCFramework
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 23
+      componentName: ''
+      isError: false
+      spanType: Exit
+      peer: '127.0.0.1:18080'
+      peerId: 0
+    - {operationName: GreeterBlocking.sayHello/client/Request/onComplete, operationId: 0,
+       parentSpanId: 2, spanId: 3, spanLayer: RPCFramework, startTime: nq 0,
+       endTime: nq 0, componentId: 23, componentName: '', isError: false,
+       spanType: Local, peer: '', peerId: 0}
+    - {operationName: GreeterBlocking.sayHello/client/Response/onClose, operationId: 0,
```

Comment:
Why do these two use different formats? I know they may be right but it seems strange.

Reply:
When the span is shorter, it is a simplified format.

Reply:
> Why do these two use different formats? I know they may be right but it seems strange.

When the span is shorter, it is a simplified format.

Reply:
Could you follow the same pattern? I prefer we keep this consistent same in all test cases.

Reply:
> Could you follow the same pattern? I prefer we keep this consistent same in all test cases.

Yes, of course.